neo/Prompt/Commands/Wallet.py

```diff
@@ -100,22 +109,76 @@ def execute(self, arguments):
 
             if PromptData.Wallet:
                 PromptData.Prompt.start_wallet_loop()
+                return
 
         else:
             print("Please specify a path")
+            return
 
     def command_desc(self):
         p1 = ParameterDesc('path', 'path to store the wallet file')
         return CommandDesc('create', 'creates a new NEO wallet address', [p1])
 
 
+class CommandWalletOpen(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        if PromptData.Wallet:
+            PromptData.close_wallet()
+
+        path = get_arg(arguments, 0)
+
+        if path:
+
+            if not os.path.exists(path):
+                print("Wallet file not found")
+                return
```

Comment:
Should return something here too.

Reply:
https://github.com/CityOfZion/neo-python/pull/726#discussion_r236978513